kingfisher_scrapy/spiders/mexico_nuevo_leon_records.py

```diff
@@ -19,13 +18,5 @@ class MexicoNuevoLeonRecords(SimpleSpider):
     def start_requests(self):
         yield scrapy.Request(
             'http://si.nl.gob.mx/siasi_ws/api/ocds/DescargarRecordPackage?usuario=undefined&servidor=undefined',
```

Comment:
Actually, it seems that we don't need to use the query params
```suggestion
            'http://si.nl.gob.mx/siasi_ws/api/ocds/DescargarRecordPackage',
```